### DISCUSSION TITLE
Connection Terminated

### DISCUSSION THREAD

**Discussion by DaddyDong** [LizardByte/community#226] [Sunshine]
I was using sunshine paired with Moonlight for a solid month with no issues then all of a sudden I started getting "Connection Terminated" within 5 sec of connected outside of the network. On the network it's fine but when I'm outside the network it disconnects. I read a few threads and put in my virtual sink & disabling my firewall to see it that corrected it and still the same issue. VPN is disabled as well.

This is the Log Report Error I get: **Initial Ping Timeout**

[2023:04:21:08:40:10]: Info: CLIENT CONNECTED
[2023:04:21:08:40:10]: Info: 
Device Description : AMD Radeon RX 5600 XT
Device Vendor ID   : 0x00001002
Device Device ID   : 0x0000731F
Device Video Mem   : 6103 MiB
Device Sys Mem     : 0 MiB
Share Sys Mem      : 16341 MiB
Feature Level      : 0x0000B100
Capture size       : 3840x1080
Offset             : 0x0
Virtual Desktop    : 3840x1080
[2023:04:21:08:40:10]: Info: Desktop resolution [3840x1080]
[2023:04:21:08:40:10]: Info: Desktop format [DXGI_FORMAT_B8G8R8A8_UNORM]
[2023:04:21:08:40:10]: Info: 
Colorspace         : DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709
Bits Per Color     : 10
Red Primary        : [0.682617,0.308594]
Green Primary      : [0.272461,0.65918]
Blue Primary       : [0.151367,0.0576172]
White Point        : [0.3125,0.329102]
Min Luminance      : 0.0491 nits
Max Luminance      : 603 nits
Max Full Luminance : 351 nits
[2023:04:21:08:40:10]: Info: Capture format [DXGI_FORMAT_B8G8R8A8_UNORM]
[2023:04:21:08:40:10]: Info: SDR color coding [Rec. 709]
[2023:04:21:08:40:10]: Info: Color range: [MPEG]
[2023:04:21:08:40:19]: Error: Initial Ping Timeout

**Comment by andrew-manger** [comment]
Same issue, haven’t found what causes the problem yet.

**Comment by psyke83** [comment]
Try to ping the host machine from the client in the same networking conditions. Ping is blocked via Windows Firewall by default unless File & Printer sharing is enabled for the current network profile, so enabling that is likely the easiest way to fix your problem, or else you may need to configure your router to allow the host to be pinged from outside the network.

In this circumstance Moonlight may close the connection because it interprets the ping timeout to mean that the actual streaming session is not working (which is not the case for you).

**Comment by panprofes** [comment]
Did anyone found resolution to this problem?

[a comment by an automated account is not shown]
